CODEOWNERS

```diff
@@ -675,7 +675,7 @@
 /subsys/fs/fcb/                           @nvlsianpu
 /subsys/fs/fuse_fs_access.c               @vanwinkeljan
 /subsys/fs/nvs/                           @Laczen
-/subsys/ipc/                              @ioannisg
+/subsys/ipc/                              @anangl
```

Comment:
Perhaps someone more appropriate could be proposed for the IPC subsystem, as I'm not really experienced in that matter. Maybe @carlocaione?

Reply:
will fix that, thanks.